new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleNginxValidation.java

```diff
@@ -288,81 +299,114 @@ public void testCreateMiiDomain() {
+          .isTrue();
+
+      // check that the ingress was found in the domain namespace
+      assertThat(assertDoesNotThrow(() -> listIngresses(domainNamespace)))
+          .as("Test ingress {0} was found in namespace {1}", ingressName, domainNamespace)
+          .withFailMessage("Ingress {0} was not found in namespace {1}", ingressName, domainNamespace)
+          .contains(ingressName);
+
+      logger.info("Ingress {0} for cluster {1} of domain {2} was found in namespace {3}",
+          ingressName, clusterName, domainUid, domainNamespace);
+    }
   }
 
   @Test
   @Order(3)
-  @DisplayName("Verify the application can be accessed through the ingress controller")
-  public void testSampleAppThroughIngressController() {
+  @DisplayName("Verify the application can be accessed through the ingress controller for each cluster in the domain")
+  public void testAppAccessThroughIngressController() {
 
-    List<String> managedServerNames = new ArrayList<>();
-    for (int i = 1; i <= replicaCount; i++) {
-      managedServerNames.add(managedServerNameBase + i);
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS; i++) {
+      String clusterName = CLUSTER_NAME_PREFIX + i;
+
+      List<String> managedServerListBeforeScale =
+          listManagedServersBeforeScale(clusterName, replicaCount);
+
+      // check that NGINX can access the sample apps from all managed servers in the cluster of the domain
+      curlCmd = generateCurlCmd(clusterName);
+      assertThat(callWebAppAndCheckForServerNameInResponse(curlCmd, managedServerListBeforeScale, 50))
+          .as("Verify NGINX can access the sample app from all managed servers in the domain")
+          .withFailMessage("NGINX can not access the sample app from one or more of the managed servers")
+          .isTrue();
     }
+  }
 
-    // check that NGINX can access the sample apps from all managed servers in the domain
-    String curlCmd = String.format("curl --silent --noproxy '*' -H 'host: %s' http://%s:%s/sample-war/index.jsp",
-        domainUid + ".test", K8S_NODEPORT_HOST, nodeportshttp);
-    assertThat(callWebAppAndCheckForServerNameInResponse(curlCmd, managedServerNames, 50))
-        .as("NGINX can access the sample app from all managed servers in the domain")
-        .withFailMessage("NGINX can not access the sample app from one or more of the managed servers")
-        .isTrue();
+  @Test
+  @Order(4)
+  @DisplayName("Verify scale each cluster of the domain in domain namespace")
+  public void testScaleClusters() {
+
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS; i++) {
+
+      String clusterName = CLUSTER_NAME_PREFIX + i;
+      int numberOfServers = 2 * i - 1;
+
+      // scale cluster-1 to 1 server and cluster-2 to 3 servers
+      logger.info("Scaling cluster {0} of domain {1} in namespace {2} to {3} servers.",
+          clusterName, domainUid, domainNamespace, numberOfServers);
+      scaleAndVerifyCluster(domainUid, domainNamespace, clusterName, replicaCount, numberOfServers);
```

Comment:
In case of scale up, verify the state of the existing servers has not changed.

Reply:
fixed